PocketKit/Sources/PocketGraph/user-defined-operations/unifiedHome.graphql

```diff
@@ -0,0 +1,42 @@
+fragment CorpusItemParts on CorpusItem {
```

Comment:
I followed the previous implementation here, which was not fetching the `SavedItem` in `ItemSummary`. I believe this was because if the `Item` was found in the database, then the related `SavedItem` would be already associated to it, does it sound right @bassrock ?

Reply:
@Gio2018 Gotcha, that way will work for 95% of users then! and fine by me. Because we only download the first 1000 items in the users list, we could recoemend an item that we didnt download.

Reply:
oh true. I think I can do a follow up PR on this then, thank you!